server/utils/trim.ts

```diff
@@ -0,0 +1,20 @@
+export default function trimForm<T extends Record<string, unknown>>(form: T): T {
+  return Object.keys(form).reduce((acc, curr) => {
```

Comment:
will see if there is anything internal that I can use - saw other services using this